src/components/EditListingDescriptionPanel/EditListingDescriptionPanel.js

```diff
@@ -24,48 +28,105 @@ const EditListingDescriptionPanel = props => {
     panelUpdated,
     updateInProgress,
     errors,
+    listingType,
   } = props;
 
   const classes = classNames(rootClassName || css.root, className);
   const currentListing = ensureOwnListing(listing);
   const { description, title, publicData } = currentListing.attributes;
+  const { manufactureYear, maxUsingTime, equipmentType } = publicData;
 
   const isPublished = currentListing.id && currentListing.attributes.state !== LISTING_STATE_DRAFT;
+  const renderMessageByListingType = () => {
+    if (listingType === LISTING_TYPE.equipment)
+      return <FormattedMessage id="EditEquipmentListingDescriptionPanel.createListingTitle" />;
+    else return <FormattedMessage id="EditListingDescriptionPanel.createListingTitle" />;
```

Comment:
we can use

const renderMessageByListingType = () => listingType === LISTING_TYPE.equipment 
? <FormattedMessage id="EditEquipmentListingDescriptionPanel.createListingTitle" /> 
: <FormattedMessage id="EditListingDescriptionPanel.createListingTitle" />;

Or

const renderMessageByListingType = listingType === LISTING_TYPE.equipment 
? <FormattedMessage id="EditEquipmentListingDescriptionPanel.createListingTitle" /> 
: <FormattedMessage id="EditListingDescriptionPanel.createListingTitle" />;